WalletWasabi.Fluent/ViewModels/Wallets/Send/ChangeAvoidanceSuggestionViewModel.cs

```diff
@@ -66,10 +66,16 @@ await foreach (var selection in selections)
 					selection,
 					tryToSign: false);
 
-				yield return new ChangeAvoidanceSuggestionViewModel(
-					transactionInfo.Amount.ToDecimal(MoneyUnit.BTC),
-					transaction,
-					wallet.Synchronizer.UsdExchangeRate);
+				var destinationAmount = transaction.CalculateDestinationAmount();
+
+				if (!amounts.Contains(destinationAmount))
```

Comment:
If you add a workaround, always add comments on why this was required. This can grab other devs attention that the root cause is not clear yet or later if somebody finds the reason this code can be removed.